.env.sample

```
@@ -129,6 +129,13 @@ REDIS_PASS=
+# Alt text generation prompt for images
+# AZURE_ALT_TEXT_PROMPT=As an AI tool specialized in image recognition, generate concise and descriptive alt text for this image. The description should be suitable for a student with a vision impairment taking a quiz. Do not include phrases like 'This is an image of...'. Provide only one concise option with no further explanation.
+
+# Temperature for alt text generation (0.0 = deterministic, 1.0 = more creative)
+# AZURE_ALT_TEXT_TEMPERATURE=0.0
+
 # Image optimization settings
 # IMAGE_MAX_DIMENSION=512
-# IMAGE_JPEG_QUALITY=85
\ No newline at end of file
+# IMAGE_JPEG_QUALITY=85
+# IMAGE_PROCESSING_CONCURRENCY=4
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)